README.cnd

```diff
@@ -1,41 +1,35 @@
 
-* not all license headers have been changed; many files are missing the license headers; binary files need licenseinfo.xml to be filled
-* building requires a number of files from the original repository - work by NetBeans C/C++ enthusiasts is needed to resolve this in one way or another
-* building requires a number of binaries to be added manually
-* external libraries/dependencies need to be reviewed and put up to Apache standard
-
-Building
---------
+      http://www.apache.org/licenses/LICENSE-2.0
 
-To build the C/C++ support, please copy the following files from the original NetBeans repository to your clone:
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
 
-* cnd.debugger.common2/src/org/netbeans/modules/cnd/debugger/common2/icons/debugger_log.png => cnd/cnd.debugger.common2/src/org/netbeans/modules/cnd/debugger/common2/icons/debugger_log.png
-* cnd.modelimpl/src/org/netbeans/modules/cnd/modelimpl/parser/CPPParserEx.java => cnd/cnd.modelimpl/src/org/netbeans/modules/cnd/modelimpl/parser/CPPParserEx.java
-* cnd.modelimpl/src/org/netbeans/modules/cnd/modelimpl/parser/CPPSymbol.java => cnd/cnd.modelimpl/src/org/netbeans/modules/cnd/modelimpl/parser/CPPSymbol.java
-* cnd.modelimpl/src/org/netbeans/modules/cnd/modelimpl/parser/cppparser.g => cnd/cnd.modelimpl/src/org/netbeans/modules/cnd/modelimpl/parser/cppparser.g
-* cnd.repository/src/org/netbeans/modules/cnd/repository/storage/data/UTF.java => cnd/cnd.repository/src/org/netbeans/modules/cnd/repository/storage/data/UTF.java
-* cnd.repository/src/org/netbeans/modules/cnd/repository/util/LongHashMap.java => cnd/cnd.repository/src/org/netbeans/modules/cnd/repository/util/LongHashMap.java
-* cnd.testrunner/src/org/netbeans/modules/cnd/testrunner/ui/CndUnitHandlerFactory.java => cnd/cnd.testrunner/src/org/netbeans/modules/cnd/testrunner/ui/CndUnitHandlerFactory.java
-* cnd.utils/src/org/netbeans/modules/cnd/utils/resources/mime-resolver-hex-based.xml => cnd/cnd.utils/src/org/netbeans/modules/cnd/utils/resources/mime-resolver-hex-based.xml
-* mercurial.remote/src/org/netbeans/modules/mercurial/remote/ui/update/ResolveConflictsExecutor.java => dlight/mercurial.remote/src/org/netbeans/modules/mercurial/remote/ui/update/ResolveConflictsExecutor.java
-* remotefs.versioning.api/src/org/netbeans/modules/remotefs/versioning/api/Scrambler.java => dlight/remotefs.versioning.api/src/org/netbeans/modules/remotefs/versioning/api/Scrambler.java
-* subversion.remote/src/org/netbeans/modules/subversion/remote/ui/update/ResolveConflictsExecutor.java => dlight/subversion.remote/src/org/netbeans/modules/subversion/remote/ui/update/ResolveConflictsExecutor.java
+-->
```

Comment:
As this file is intended for users, I would use the short version of the license header here and as this is neither markdown, nor html, I would not use comment delimiters.

```
Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements; and to You under the Apache License, Version 2.0.
```